processors/config/anaSvtTiming_cfg.py

```diff
@@ -0,0 +1,75 @@
+import HpstrConf
+import sys
+import os
+import baseConfig as base
+
+
+options = base.parser.parse_args()
+
+
+# Use the input file to set the output file name
+infile = options.inFilename
+outfile = options.outFilename
+
+print('Input file: %s' % infile)
+print('Output file: %s' % outfile)
+
+p = HpstrConf.Process()
+
+p.run_mode = 1
+#p.max_events = 1000
+
+# Library containing processors
+p.add_library("libprocessors")
+
+###############################
+#          Processors         #
+###############################
+
+recoana = HpstrConf.Processor('recoana', 'SvtTimingProcessor')
+
+###############################
+#   Processor Configuration   #
+###############################
+#SvtTiming
+recoana.parameters["debug"] = 0
+recoana.parameters["anaName"] = "svtHitTimingAna"
+recoana.parameters["trkColl"] = "KalmanFullTracks"
+#recoana.parameters["trkrHitColl"] = "RotatedHelicalTrackHits"
+recoana.parameters["rawHitColl"] = "SVTRawHitsOnTrack_KF"
+recoana.parameters["trkrHitColl"] = "SiClustersOnTrack"
+recoana.parameters["fspColl"] = "FinalStateParticles_KF"
+recoana.parameters["ecalHitColl"] = "RecoEcalHits"
+recoana.parameters["ecalClusColl"] = "RecoEcalClusters"
+recoana.parameters["histCfg"] = os.environ['HPSTR_BASE']+'/analysis/plotconfigs/reco/svtTiming-ecalRef.json'
+recoana.parameters["mcHistoCfg"] = os.environ['HPSTR_BASE']+'/analysis/plotconfigs/mc/basicMC.json'
+#recoana.parameters["timingCalibFile"]="/sdf/group/hps/users/mgraham/TimingCorrections/physrun2021/run14507_calib.txt"
+#recoana.parameters["timingCalibFile"]="/sdf/group/hps/users/mgraham/TimingCorrections/physrun2021/svtTiming_ecalRef_run_14495_preCal_calib.txt"
+recoana.parameters["timingCalibDir"]="/sdf/group/hps/users/mgraham/TimingCorrections/physrun2021/perRunTiming"
```

Comment:
Can we change this to a path relative to HPS_BASE or PWD? The other option would be to make it a command line option and I would be okay with this being the default.